### PR TITLE
ARM: tegra: Add device-tree for ASUS Transformer Prime TF201

### PR DESCRIPTION
**Pull request by clamor95**
We were working, were working, and finally, oh GOD, we made it boot and a tonn of staff working...on TF201.

@digetx may you squash this commit into my previous commit with same name? Use name of original and description of this one.

Hope tree is good enough for this stage. If doesn't brake build this time. [Check TF201 PmOS wiki page)](https://wiki.postmarketos.org/wiki/ASUS_Transformer_Prime_(asus-tf201))

### CODE REVIEW

**Review by digetx — COMMENTED** [review]
Congratulations with getting it to work! Could you please should output of `dmesg`, `/sys/kernel/debug/clk/clk_summary`,  `/sys/kernel/debug/regulator/regulator_summary` and `/sys/kernel/debug/pm_genpd/pm_genpd_summary`?

File: arch/arm/boot/dts/tegra30-asus-tf201.dts [hunk 1]
```diff
@@ -180,15 +150,11 @@
 		};
 	};
 
-	/* internal SDIO WiFi */
-	mmc@78000400 {		/* sdmmc3 */
+	/* internal SDIO WiFi sdmmc3 */
+	sdhci@78000400 {
```
[the remark this answers is not in the document]
Comment (digetx) [reply arch/arm/boot/dts/tegra30-asus-tf201.dts:154]: I'll fix up it locally.

**Comment by clamor95** [comment]
@digetx you may find all logs you asked on my GDrive https://drive.google.com/folderview?id=1Lp3krIewehOjtn1jgEzcWdKpwoZ2Iifv Any help is welcome!

**Comment by digetx** [comment]
The logs look okay in general, you'll need to fix USB host and figure out what causes the timeouts. For USB you probably need to set a proper VBUS-supply regulator in the device-tree. 

I added https://github.com/grate-driver/linux/commit/ed2ba04545026bb080666fa2c8bb0683faee2cb6 which was used for debugging pinmux problems on Nexus 7 and A500.

Grab output of the following files from grate-kernel and from downstream kernel, show it to me:

 - `/sys/kernel/debug/tegra_pinmux`
 - `/sys/kernel/debug/tegra_pinmux_drive`
 - `/sys/kernel/debug/tegra_gpio`

**Comment by clamor95** [comment]
@digetx eventually Ion Aggoria fixed USB host. Original 3.1 kernel has no regulators for that (so no control gpios are available). I analyzed trees of grouper, picasso and ouya and found that vbus can be shipped directly by 5v regulator, since all additional regs descend from 5v.

**Comment by digetx** [comment]
Thanks, I answered to Meowin on #tegra irc.

**Pull request merged**